controlpanel/kubeapi/views.py

```diff
@@ -1,30 +1,16 @@
 from functools import wraps
```

Comment:
I know it's not connected to this PR, but this import seems unused

Reply:
Nice catch 👍